pkg/blob/nodeserver.go

```diff
@@ -151,15 +153,19 @@ func (d *Driver) NodeStageVolume(ctx context.Context, req *csi.NodeStageVolumeRe
 	}
 
 	if strings.TrimSpace(blobStorageEndPoint) == "" {
+		blobStorageEndPoint = "core.windows.net"
```

Comment:
it's called `storageEndpointSuffix`, let's use official term:
```
StorageEndpointSuffix:        "core.windows.net",
```

Reply:
done